pkg/replica/replica.go

```diff
@@ -221,6 +221,10 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 		}
 	}
 
+	if err := r.checkVolumeMetaFile(); err != nil {
```

Comment:
If the volume.meta is empty, doesn't it mean this function will fail at the above step `exists, err := r.readMetadata()` earlier before running this recovery? If yes, why this will work?

Reply:
Because file which is empty had been removed at (s *Server) Status()(State, Info)

Reply:
Suggest recovering the file before `readMetadata`, so you don't need to change other places.

Reply:
Moved. still need to check `io.EOF` caused by empty meta file at server.Status() but it will not delete the file, handle empty file in `tryRecoverVolumeMetaFile`